docs/data/joy/main-features/color-inversion/color-inversion.md

```diff
@@ -41,88 +36,52 @@ The color inversion feature is only available for `soft` and `solid` variants be
-To enable it, set `disablePortal` to true on the `slotProps`.
-
-:::info
-The popup slot of the Select component has `disablePortal` set to true by default.
-:::
+Color inversion does not affect the popup slot of the Autocomplete, Menu, and Tooltip components by default.
+To enable it, on the `slotProps` prop of these components, set `disablePortal` to true.
 
 {{"demo": "ColorInversionPopup.js"}}
 
-### Skip color inversion on a child
-
-To skip color inversion on a specific child, set `data-skip-inverted-colors` attribute to the component.
+### Turning it off on a specific child
 
-The component with `data-skip-inverted-colors` and its children will be excluded from the color inversion.
+Add the `data-skip-inverted-colors` attribute to the child component you want to turn color inversion off within an inverted container.
 
 {{"demo": "ColorInversionSkip.js"}}
 
 ### Apply color inversion to any parent
 
-Use `applySolidInversion` or `applySoftInversion` utilities to apply color inversion to any parent component.
-
-They are used internally by the surface components, e.g. [Card](/joy-ui/react-card/#inverted-colors), when `invertedColors` prop is set to true.
-
 ```js
 import { applySolidInversion, applySoftInversion } from '@mui/joy/colorInversion';
 ```
 
-Example usage for `sx` prop and `styled` API:
-
-- `sx` prop
-
-  ```js
-  import { applySolidInversion } from '@mui/joy/colorInversion';
-
-  <Box
-    sx={[
-      (theme) => ({
-        display: 'flex',
-        alignItems: 'center',
-        background: theme.vars.palette.neutral[900],
-      }),
-      applySolidInversion('neutral'),
-    ]}
-  >
-    …
-  </Box>;
-  ```
-
-- `styled` API
-
-  ```js
-  import { styled } from '@mui/joy/styles';
-  import { applySoftInversion } from '@mui/joy/colorInversion';
-
-  const StyledBox = styled(Box)(
-    ({ theme }) => ({
-      display: 'flex',
-      alignItems: 'center',
-      ...theme.variants.soft.primary,
-    }),
-    applySoftInversion('primary'),
-  );
-  ```
+Use the `applySolidInversion` or `applySoftInversion` utilities to add color inversion to any parent component.
+The Card component, for example, already uses it internally if containing the `invertedColors` prop.
+
+The examples below show how to use it with either the `sx` prop or the `styled` API:
+
+#### With the sx prop
 
 {{"demo": "ColorInversionAnyParent.js"}}
 
+#### With the styled API
+
+{{"demo": "ColorInversionAnyParentStyled.js"}}
+
 ## How it works
 
-When `invertedColors` is set to true or the utility is used on the parent component, a set of CSS variables is applied to it. There is no [React context](https://react.dev/learn/passing-data-deeply-with-context) involved in this feature.
+Color inversion adds CSS variables to the component using the `invertedColors` prop or the apply utilities. There's no [React context](https://react.dev/learn/passing-data-deeply-with-context) involved in this feature.
```

Comment:
Why does it matter that there's no React context involved? Is it just because it's better to do things with CSS rather than JS when possible? Or is there some other reason?

Reply:
I would assume it's a simplicity value — this being pulled off without JavaScript seems like a value. But it's definitely something @siriwatknp can better inform us! 🤙